pdb2sql/pdb2sqlcore.py

```diff
@@ -410,54 +433,58 @@ def get(self, atnames, **kwargs):
 
         return data
 
-    def update(self, attribute, values, **kwargs):
+    def update(self, columns, values, **kwargs):
         '''Update the database.
 
         Args:
-            attribute (str) : string of attribute names eg. ['x','y,'z'], 'xyz', 'resSeq'
+            columns (str): names of column to update, e.g. "x,y,z".
+            values (np.ndarray): an array of values that corresponds
+                        to the number of columns and atoms selected.
+            **kwargs: selection arguments,
+                eg: name = ['CA', 'O'], chainID = ['A'],
+                or no_name = ['CA', 'C'], no_chainID = ['A'].
+        Examples:
+            >>> values = np.array([[1.,2.,3.], [4.,5.,6.]])
+            >>> self.db.update("x,y,z", values=values, resName='MET', name=['CA', 'CB'])
+        '''
+        # check arguments format
+        valid_colnames = self.get_colnames()
 
-            values (np.ndarray) : an array of values that corresponds
-                                  to the number of attributes and atoms selected
+        if not isinstance(columns, str):
+            raise TypeError("argument columns must be str")
 
-            **kwargs : selection arguments eg : name = ['CA','O'], chainID = 'A', no_name = ['H']
-        '''
+        if columns != '*':
+            for i in columns.split(','):
+                if i not in valid_colnames:
+                    raise ValueError(
+                        f'Invalid column name {i}. Possible names are\n'
+                        f'{self.get_colnames()}')
 
         # the asked keys
         keys = kwargs.keys()
 
-        # check if the column exists
```

Comment:
same here, why did you remove that part ?

Reply:
Same as above.